misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickStepButton.jsx

```diff
@@ -97,24 +82,31 @@ class PickStepButton extends PureComponent {
   }
 }
 
+const mapStateToProps = (state, ownProps) => {
+  const { wfProcessId, activityId, lineId, stepId } = ownProps;
+
+  return {
+    stepState: state.wfProcesses_status[wfProcessId].activities[activityId].dataStored.lines[lineId].steps[stepId],
```

Comment:
why not get it from the caller component via props instead of fetching it directly from the state?

Reply:
We can and we should so changed that. It was a leftover when we had this parent `StepButton` and we didn't know what props will be needed in the wrapped component.